cdm-test/src/test/java/ucar/nc2/ncml/TestOffAggNewSync.java

```diff
@@ -37,14 +41,14 @@ public class TestOffAggNewSync {
   @Ignore("file in use - testing artifact")
```

Comment:
any idea what this ignore is for..?

Reply:
not really, I wanted to fix up this class as well but wasn't sure how it was different from the other one and when I un-ignored it that I got errors like "not a CDM dataset" for that file.

So I only cleaned up the other one (`TestOffAggUpdating`) and moved the helper functions this class needed here.